src/components/CreateArticleForm.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import {
+  Button,
+  Container,
+  Form,
+  Header,
+  Input,
+  TextArea,
+  Message,
+  } from "semantic-ui-react";
+
+import { useDispatch, useSelector } from "react-redux";
+import { CreateArticle } from "../modules/CreateArticle";
+
+function CreateArticleForm() {
+  const dispatch = useDispatch();
+  const { createArticleMessage, errorMessage } = useSelector((state) => state);
```

Comment:
We can change (state) to state just for readability

Reply:
FIXED!